src/sql-parser/src/parser.rs

```diff
@@ -3388,13 +3388,15 @@ impl<'a> Parser<'a> {
     }
 
     fn parse_index_option_name(&mut self) -> Result<IndexOptionName, ParserError> {
-        self.expect_keywords(&[LOGICAL, COMPACTION, WINDOW])?;
-        Ok(IndexOptionName::LogicalCompactionWindow)
```

Comment:
Nikhil checked and he thinks we don't need to do this since the flag in LD has never been turned on for anyone -- https://materializeinc.slack.com/archives/CTESPM7FU/p1706909326394939?thread_ts=1706908723.778169&cid=CTESPM7FU

Reply:
Works for me! Thanks for double checking!